packages/disploy/src/structs/managers/StructureManager.test.ts

```diff
@@ -19,7 +19,7 @@ class ExampleClass extends Base {
 	public name: string;
 
 	public constructor(_: null, raw: ExampleData) {
-		super(null!);
+		super(null!, raw);
```

Comment:
While this would probably be a good idea to let the user access the raw objects; it is out of the scope for this PR.